packages/config/test/unit/index_spec.js

```diff
@@ -146,4 +146,30 @@ describe('src/index', () => {
       })
     })
   })
+
+  describe('.validateNoReadOnlyConfig', () => {
+    it('returns an error if validation fails', () => {
+      const errorFn = sinon.spy()
+
+      configUtil.validateNoReadOnlyConfig({ chromeWebSecurity: false }, errorFn)
+
+      expect(errorFn).to.have.callCount(1)
+    })
+
+    it('does not return an error if validation succeeds', () => {
+      const errorFn = sinon.spy()
+
+      configUtil.validateNoReadOnlyConfig({ requestTimeout: 1000 }, errorFn)
+
+      expect(errorFn).to.been.callCount(0)
+    })
+
+    it('does not return an error if configuration is a non-Cypress config option', () => {
+      const errorFn = sinon.spy()
+
+      configUtil.validateNoReadOnlyConfig({ foo: 'bar' }, errorFn)
+
+      expect(errorFn).to.been.callCount(0)
+    })
```

Comment:
Can we also assert that the argument passed to the error function is the expected option?

Reply:
While you're at it, `expect(errorFn).to.have.callCount(0)` or `expect(errorFn).not.to.have.beenCalled` 😬

Reply:
Should be updated!